packages/js/src/decorator/gutenberg.js

```diff
@@ -144,6 +144,18 @@ export function isAnnotationAvailable() {
  */
 export function getYoastmarkOffsets( marked ) {
 	let startMarkIndex = marked.indexOf( START_MARK );
+
+	// Checks if the start mark is single quoted.
+	// Note: if isMaybeSingleQuoted is true, this does nessarily mean that the start mark is single quoted.
+	// It could also be that the start mark doesn't occur at all in startMarkIndex.
+	// In that case, startMarkIndex will be -1 during later tests.
+	const isMaybeSingleQuoted = startMarkIndex >= 0;
```

Comment:
Question for CR: not sure what would be a good name for `isMaybeSingleQuoted`. Suggestions would be welcome.

Reply:
The reason I went with Maybe is because of the following note
`// Note: if isMaybeSingleQuoted is true, this does necessary mean that the start mark is single quoted.`

if `startMarkIndex >= 0`, this means that the doublequoted start mark is not present. This could either mean that the start mark is single quoted, or that no start mark occurs.

Reply:
Sorry, I think I misunderstood the comment then. 
So, there are three cases: no start mark, single quote, and double quote. `startMarkIndex >= 0` is true for single quote and false for double quote, but is it supposed to be true or false for no start mark occurs? 
But, if it is also true for no start mark, wouldn't it be appropriate to call the variable `isNotDoubleQuoted`?

Reply:
good suggestion. Will apply.